docs/tutorial/Basic/tensor.ipynb

```diff
@@ -0,0 +1,1279 @@
+{
```

Comment:
<p>remove extra line at the end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>